baselines/flwr_baselines/flwr_baselines/publications/leaf/femnist/model.py

```diff
@@ -55,7 +55,7 @@ def train(
     verbose: bool = False,
 ) -> Tuple[float, float, Optional[float], Optional[float]]:
     """Train a given model with CrossEntropy and SGD (or some version of it
-    like batch-SGD).
+    like batch- SGD).
```

Comment:
Why did it add a space here? I think the comment should be split into a single-line first sentence and a possibly longer second sentence.

Reply:
@danieljanes I don't know, but given this is a deprecated code, I'd suggest skipping to put too much effort into it. I made a slight adjustment. I would assume that @jafermarq will have a look anyway when migrating the code into the new baseline format.

Reply:
yes